docs/user/developers/integration-apis.rst

```diff
@@ -132,6 +132,25 @@ providers.
 - Pricing Model: Free up to 20k requests, Pricing tiers
 - Documentation: https://nownodes.io/documentation
 
+Tokenview
+--------
+
+.. image:: img/tokenview.png
+   :width: 200px
+   :align: right
+   :target: https://services.tokenview.io
+
+https://services.tokenview.io
+
+Tokenview provides a wealth of DASH APIs, easy to perform cryptocurrency 
+payment applications and develop Dash dApps. Monitor & Alert is served 
+to receive balance changes in real time. 
```

Comment:
Does this slight re-wording look okay to you @Tokenview? (thanks @strophy for the input)
```suggestion
Tokenview provides a Dash API, making it easy for developers to create
cryptocurrency payment applications and develop Dash dApps. Monitoring
and alerts features are available to notify applications of balance changes
in real time.
```

Reply:
Yes, that's good. Thanks for helps us for editing.